libs/blocks/modal/decorators/langnav.js

```diff
@@ -0,0 +1,37 @@
+import { getConfig } from '../../../utils/utils.js';
+
+function handleEvent(prefix, link, config) {
+  document.cookie = `international=${prefix};path=/`;
```

Comment:
is it ok to blow away any existing cookies?

Reply:
I'd like to say yes, but unfortunately no. As I've understood it, up until now adobe.com has set cookies for the georouting and other code also checks what value is in the international cookie. I'm still setting the cookie so this old code keeps working properly.